Sources_v3/StreamChatUI/ChatChannel/ChatChannelMessageComposerView.swift

```diff
@@ -4,165 +4,173 @@
 
 import UIKit
 
-open class ChatChannelMessageComposerView<ExtraData: UIExtraDataTypes>: UIInputView {
+open class ChatChannelMessageComposerView<ExtraData: UIExtraDataTypes>: UIInputView,
+    UIConfigProvider,
+    Customizable,
+    AppearanceSetting {
     // MARK: - Properties
     
-    public var buttonHeight: CGFloat = 20
+    public var attachmentsViewHeight: CGFloat = .zero
+    public var stateIconHeight: CGFloat = .zero
     
-    public let uiConfig: UIConfig<ExtraData>
-        
     // MARK: - Subviews
 
     public private(set) lazy var container = ContainerStackView().withoutAutoresizingMaskConstraints
     
     public private(set) lazy var replyView: UIView = .init()
     
-    public private(set) lazy var attachmentsView: MessageComposerAttachmentsView<ExtraData> = .init(uiConfig: uiConfig)
+    public private(set) lazy var attachmentsView: MessageComposerAttachmentsView<ExtraData> = {
+        uiConfig.messageComposer.attachmentsView.init().withoutAutoresizingMaskConstraints
+    }()
```

Comment:
```suggestion
    public private(set) lazy var attachmentsView = uiConfig
      .messageComposer
      .attachmentsView.init()
      .withoutAutoresizingMaskConstraints
```

Reply:
Yeah, I like how it looks, we should maybe add this style to `Notion` docs if everyone agree.

Reply:
@mantergo I think it's ok to have it _just_ in the code :))